s/frontend/components/common/five-stars/namaka-review.ts

```diff
@@ -0,0 +1,26 @@
+import {renderFiveStarRating, stateBuddy} from './render-five-star-display.js';
+import fiveStarRatingCss from './render-five-star-display.css.js';
+import {mixinStyles} from '../../../framework/mixins/mixin-styles.js';
+import {html, LitElement} from 'lit';
+
+@mixinStyles(fiveStarRatingCss)
+export class NamakaReview extends LitElement {
+	render() {
+
+		customElements.whenDefined('namaka-review').then(() => {
+
+			let el: HTMLElement = document
+				.querySelector('namaka-review')
+				?.shadowRoot?.querySelector('.test')!;
+			
+			const fivestar = stateBuddy(
+				el,
```

Comment:
we try to avoid cryptic shotnames like `el`. it shouldn't even be `element`, because that wouldn't describe what this element is about. certainly `test` isn't a good name for a final product...